pyspecProcScripts/load_data.py

```diff
@@ -838,6 +838,7 @@ def proc_field_sweep_v2(s):
     "ag_CPMG_strob": proc_bruker_CPMG_v1,
     "ag_T1CPMG_2h": proc_bruker_T1CPMG_v1,
     "chirp": proc_capture,
+    "GDS_capture_v1": proc_capture,
```

Comment:
I don't think that you want this b/c you don't want to do anything to it, and don't need to pass the lookup table to find file.

Reply:
the way that you have the code written above (which is correct) this will never get used